checker/tests/initialization/TestPolyInitialized.java

```diff
@@ -0,0 +1,36 @@
+import org.checkerframework.checker.initialization.qual.Initialized;
+import org.checkerframework.checker.initialization.qual.NotOnlyInitialized;
+import org.checkerframework.checker.initialization.qual.PolyInitialized;
+import org.checkerframework.checker.initialization.qual.UnknownInitialization;
+
+public class TestPolyInitialized {
+
+    @NotOnlyInitialized String testStr;
+
+    String test = "test";
+
+    TestPolyInitialized(@UnknownInitialization String str) {
+        this.testStr = str;
```

Comment:
You can also call `identity` here and ensure you get the expected results.

Reply:
Done